src/api/index.js

```diff
@@ -18,6 +19,23 @@ const { getPluginPath } = require('../utils/paths');
 const config = new Conf();
 
 /**
+  * Lists plugins & themes with the keyword 'dext-plugin' or 'dext-theme' on npm
+  *
+  * @param {String} plugin - The name of the plugin/theme
+  * @return {String} - An array with packages matching the search
+  */
+const search = (searchTerm) => new Promise((resolve, reject) => {
```

Comment:
This should return a `Promise` that resolves into an `Array` of valid package names like so:

```js
[
  'dext-github-plugin',
  'dext-giphy-plugin',
  'dext-hackernews-plugin',
  // ..etc
]
```